src/configs/nuage/scripts/pps-simulator/config.ini

```diff
@@ -12,6 +12,7 @@ npm_count: 20
 perf_mon_count: 2
 duc_count: 2
 #Start and EndTime Format: yyyy/mm/dd/h/m/s
-start_time: 2017/03/29/0/0/0
-end_time: 2017/03/29/0/0/30
+start_time: 2017/09/14/0/0/0
+end_time: 2017/09/14/0/30/0
```

Comment:
This variable should not be used at all. It should always be now to now-24hrs to test things out.